stdlib/public/core/StringGuts.swift

```diff
@@ -386,13 +386,16 @@ extension _StringGuts {
   }
 
   @_versioned
-  @inline(never) // Hide CF dependency
+  @_inlineable
+  //@inline(never) // Hide CF dependency
```

Comment:
Why is this safe to comment out?

Reply:
Ah, this was part of a different optimization attempt I was making. It's safe to comment out as the reference to CFStringGetLength was changed to a different inline(never) call. I should just drop this from this PR though. It's more so an attempt to expose the triviality of ARC.

Reply:
Ah, this was part of a different optimization attempt I was making. It's safe to comment out as the reference to CFStringGetLength was changed to a different inline(never) call. I should just drop this from this PR though. It's more so an attempt to expose the triviality of ARC.